explorer/src/components/Filters/Filters.tsx

```diff
@@ -132,12 +134,13 @@ export const Filters = () => {
               Clear
             </Button>
           }
-          sx={{ width: 300 }}
+          sx={{ width: 300, alignItems: 'center' }}
         >
-          Filters applied
+          {mixnodes?.data?.length} mix nodes matched your criteria
         </Alert>
       </Snackbar>
-      <IconButton size="large" onClick={handleToggleShowFilters}>
+      <IconButton size="large" sx={{ borderRadius: 1 }} onClick={handleToggleShowFilters}>
+        <Typography sx={{ marginRight: 1 }}>Advanced filtering</Typography>
```

Comment:
Use a mui button here and pass in end icon. Something like this 
`<Button endIcon={<Tune />} onClick... variant="outlined" color="inherit">Advanced filters</Button>`

Reply:
Change wording to "Advanced filters"

Reply:
![image](https://user-images.githubusercontent.com/13712075/182863044-85cd62c3-5b9f-4e0f-af05-32bdbd48eb5c.png)

Make this a little wider so that the content is all on one line